log-manager/src/main/java/io/airlift/log/StaticFormatter.java

```diff
@@ -60,8 +60,15 @@ public StaticFormatter()
     public String getSystemContextSummary()
     {
         return new StringWriter()
-                .append("System Context Summary")
-                .append("\t")
+                .append(TIMESTAMP_FORMATTER.format(ZonedDateTime.now()))
+                .append('\t')
+                .append(Level.INFO.name())
+                .append('\t')
+                .append(Thread.currentThread().getName())
+                .append('\t')
+                .append(StaticFormatter.class.getCanonicalName())
```

Comment:
You could just use "SystemContext" for the logger name, and then you wouldn't have to repeat it in the log messsage.  This is just an idea... not sure if it is better or worse.

Reply:
That makes sense